pkg/docker/cockpit-atomic-storage

```diff
@@ -428,8 +424,10 @@ def cmd_monitor():
         if len(r) > 0:
             if mon.stdout.readline().decode().startswith("UDEV  "):
                 got_event = True
+        # we want to periodically all get_info() to update the usage
+        # information even without uevents
         # XXX - skip udev/lvm stuff on pure timeouts
-        info = get_info()
+        info = get_info(got_event)
```

Comment:
The parameter of the function is called `got_uevent` - is this different on purpose? I don't see that being used right now. If the tests pass, then maybe this path isn't covered by tests and needs a new one.